gossip/groups.py

```diff
@@ -71,6 +72,16 @@ def set_strict(self, strict=True):
                 child.validate_strict()
         self._strict = strict
 
+    def should_not_be_muted(self):
+        self._can_be_muted = False
+
+    def can_be_muted(self):
+        if self._can_be_muted is not None:
+            return self._can_be_muted
```

Comment:
Should it be allowed to allow muting on a specific hook of a group which cannot be muted? Not sure...